README.md

```diff
@@ -1,4 +1,6 @@
 # libevent-rs
+[![](https://meritbadge.herokuapp.com/libevent)](https://crates.io/crates/libevent)
+[![Released API docs](https://docs.rs/libevent/badge.svg)](https://docs.rs/libevent)
```

Comment:
Another useful badge is the MSRV: ![Minimum rustc version](https://img.shields.io/badge/rustc-1.42+-lightgray.svg)

Reply:
Made #12 to track this. As a sidenote, while looking for authoritative references for this I found https://github.com/rust-lang/rust/issues/65262 which seeks to allow declaring MSRV directly in the `Cargo.toml`, pretty cool.